regression/index.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var assert = require('assert');
+var env = require('./env');
+var gcloud = require('../lib');
+var uuid = require('node-uuid');
+
+// Test used to confirm we can perform a successful API operation.
+function canConnect(config, callback) {
+  var storage = gcloud.storage(config);
+
+  var BUCKET_NAME = 'gcloud-test-bucket-temp-' + uuid.v1();
+  storage.createBucket(BUCKET_NAME, function(err, bucket) {
```

Comment:
Can we pick a less rate limited request? lol. And perhaps one that doesn't POST so we don't have two steps to it that could fail half way through and leave buckets laying around.

My thought is list buckets.